presc/copies/copying.py

```diff
@@ -129,9 +129,7 @@ def generate_synthetic_data(self, **k_mod_sampling_parameters):
             # Call balancer generating function with sampling parameters
             # (sampling_balancer returns a pandas dataframe)
             X_generated = sampling_balancer(
-                numerical_sampling=self.sampling_function,
-                original_classifier=self.original,
-                **k_sampling_parameters_gen
+                original_classifier=self.original, **k_sampling_parameters_gen
```

Comment:
What is this change for? This breaks the enforce_balance functionality, as `numerical_sampling` is still a required param in `sampling_balancer()`

Relatedly, for the `enforce_balance == False` case, the mixed data notebook breaks since the `numerical_sampling` param is passed twice.

Reply:
I've changed the sampling parameter names and made them mandatory in the mixed_ function.

> Looks like idx2numpy is not available for conda, which is causing the CI failure. You'd need to put it under the `pip` section in the environment file.

I did this, but it doesn't seem to be enough.